packages/report-server/src/types.ts

```diff
@@ -33,6 +33,7 @@ export interface ReportConfig {
     dataGroups?: string[];
```

Comment:
Not your code, but might be nice to add `aggregations?: (string | Record<string, unknown)[];` to this type 👍